recipe/meta.yaml

```diff
@@ -10,13 +10,14 @@ source:
   sha256: 6ff5f3180870836cae40f06fa10419f557208175f13ad7bc26caa77beb1f6e02
 
 build:
-  number: 0
-  script: pip install .
```

Comment:
Instead, it should `pip install --no-deps .`

Reply:
Good point! I amended the commit and re-pushed the branch. Let me know if there is anything else I can do to help get this accepted. Thanks!